ray-on-gke/kuberay-tpu-webhook/main.go

```diff
@@ -24,9 +26,10 @@ type slice struct {
 type patch map[string]any
 
 var (
-	tpuResourceName = corev1.ResourceName("google.com/tpu")
 	certPath = "/etc/kuberay-tpu-webhook/tls/tls.crt"
 	keyPath = "/etc/kuberay-tpu-webhook/tls/tls.key"
+	tpuResourceName = corev1.ResourceName("google.com/tpu")
+	headlessServiceName = "tpu-worker-svc"
```

Comment:
Looks like you are extracting the Ray Cluster name in line 248, so can we base this name from that? 

That way if a user has multiple Ray clusters, each one can have a unique tpu-worker-svc based on the cluster name, and the same webhook can be reused.

Reply:
Done.